src/components/Accordion.js

```diff
@@ -0,0 +1,93 @@
+const menuOpen = "collapse show";
+const buttonOpenAll = "Open All";
+const buttonCloseAll = "Close All";
+
+if (!String.prototype.includes) {
+  String.prototype.includes = function(search, start) {
+    "use strict";
+    if (typeof start !== "number") {
+      start = 0;
+    }
+
+    if (start + search.length > this.length) {
+      return false;
+    } else {
+      return this.indexOf(search, start) !== -1;
+    }
+  };
+}
+
+document.addEventListener(
+  "click",
+  clickEvent => {
+    const { target } = clickEvent;
+
+    if (target && target.className === "dg_allitems") {
+      AllMenuItemsAction(target);
+    } else if (target) {
+      MenuAction(target);
+    } else {
+      return;
+    }
+  },
+  false
+);
+
+function MenuAction(element) {
+  var mainDiv = element.parentElement.parentElement;
+  var menuItems = mainDiv.getElementsByClassName("multi-collapse");
+  var childDiv = element.nextElementSibling;
+  var menuID = childDiv.id;
+
+  for (i = 0; i < menuItems.length; i++) {
+    if (menuID !== menuItems[i].id) {
+      menuItems[i].className = mainDiv.className.includes(
```

Comment:
since you are repeaeating menuItems[i] several times, is worth creating a variable for it. something like

```js
const menuItem = menuItems[i];
```

Reply:
oh crap, that's because I had var i = 0 above then took it out and forgot to add var to the loop. Thanks for the catch on that.

Reply:
done